openedx/FeedbackXBlock
# feat: add instructor dashboard integration

### Description

This PR set an instructor dashboard integration with a report of the answers of every student and an aggregate report:

### Testing instructions

1. In your environment add the following setting:
```python
      OPEN_EDX_FILTERS_CONFIG = {
        "org.openedx.learning.instructor.dashboard.render.started.v1": {
          "fail_silently": False,
          "pipeline": [
            "feedback.extensions.filters.AddFeedbackTab",
          ]
        },
      }
     FEATURES["ENABLE_FEEDBACK_INSTRUCTOR_VIEW"] = True
```
2. In your course add a feedback component.
3. Go to the course and answer the feedback component as a student or as an instructor.
4. Go to the instructor dashboard and verify the following table appears with the feedback information: http://local.overhang.io:8000/courses/course-v1:demo+demo+demo/instructor#view-feedback_instructor
![image](https://github.com/openedx/FeedbackXBlock/assets/33465240/ef277d3e-fc3a-4d24-a19c-08d50ff30485)
